src/excmds.ts

```diff
@@ -1472,6 +1473,18 @@ export async function credits() {
     tabopen(creditspage)
 }
 
+/** Opens the current page in Firefox's reader mode.
+ * You currently cannot use Tridactyl while in reader mode.
+ */
+//#background
+export async function reader() {
+    document.body.prepend("HELLO WORLD")
+    console.log("goodbye moon")
```

Comment:
The background console doesn't show all objects. You need a background console in the extension context: `about:debugging`, This Firefox, click Inpsect next to Tridactyl, then click console.